src/org/openstreetmap/josm/data/validation/tests/TagChecker.java

```diff
@@ -687,6 +696,33 @@ public void check(OsmPrimitive p) {
                 }
             }
         }
+
+        if(checkRegions) {
+            TaggingPreset preset = new TaggingPreset();
+            if(preset.regions()!= null) {
+                LatLon center = null;
+                if (p instanceof Node) {
+                    center = ((Node) p).getCoor();
+                } else if ((p instanceof Way) || (p instanceof Relation)) {
```

Comment:
```suggestion
                } else {
```
All other possibilities have `getBBox`, which is why you didn't have to cast in the next statement.